src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    protected static final String TODO_ERROR = "☹ OOPS!!! The description of a todo cannot be empty.";
+    protected static final String DEADLINE_DESCRIPTION_ERROR = "☹ OOPS!!! The description of a deadline cannot be empty.";
+    protected static final String EVENT_DESCRIPTION_ERROR = "☹ OOPS!!! The description of a event cannot be empty.";
+    protected static final String DEADLINE_DATE_ERROR = "☹ OOPS!!! Please add a date for this deadline task";
+    protected static final String EVENT_DATE_ERROR = "☹ OOPS!!! Please add a date for this event task";
+    protected static final String DONE_WRONG_NUMBER_ERROR = "☹ OOPS!!! Please provide a valid task number";
+    protected static final String DONE_ERROR = "☹ OOPS!!! Please input the command correctly (e.g done 1)";
+    protected static final String GENERAL_ERROR = "☹ OOPS!!! I'm sorry, but I don't know what that means :-(";
+
+
+
+    public static void main(String[] args) throws DukeExceptions {
+
+        //creating list of task
+        ArrayList<Task> taskings = new ArrayList<>();
+
+        File file = new File("/Users/desmond/ip/data");
```

Comment:
You could consider storing the directory path in a constant variable, this makes the mode more readable and is easier to change values if needed (such as the repeated directory path in the line below)